### PR TITLE
CRUD operations for users

### PR DESCRIPTION
**Pull request by fistonhn**
#### What does this PR do?
-create user
-get all users
-get one user
-update user
-delete user
#### Description of Task to be completed?
CRUD operations for users
#### How should this be manually tested?
by using postman
#### Any background context you want to provide?
N/A
#### What are the relevant Trello boards stories?
https://trello.com/c/Xuo1PoSm/108-crud-operations-for-user
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A

### CODE REVIEW

**Review by Rwandarushya — CHANGES_REQUESTED** [review]
Please work on the your travis integration

File: .travis.yml [hunk 1]
```diff
@@ -20,4 +23,4 @@ notifications:
 
   email: false
 
-sudo: false
+sudo: false
```
Comment (Rwandarushya) [inline .travis.yml:26]: Add new line to your end of file to not break eslint rules

**Review by Rwandarushya — COMMENTED** [review]
I commented on your .travis.yml file there is one eslint error, you should check on it

File: .travis.yml (hunk 1 above)
Comment (fistonhn) [reply .travis.yml:26]: thank you marius

**Pull request closed without merging**